openmetadata-ui/src/main/resources/ui/src/utils/TagsUtils.tsx

```diff
@@ -555,3 +555,15 @@ export const getTagAssetsQueryFilter = (fqn: string) => {
     return `(tags.tagFQN:"${fqn}")`;
   }
 };
+
+export const getTagImageSrc = (iconURL: string) => {
+  if (!iconURL) {
+    return '';
+  }
+
+  if (iconURL.startsWith('http') || iconURL.startsWith('data:image')) {
```

Comment:
@karanh37 , using regex will be better here and also can we handle the https as well?